### ISSUE TITLE
Remove Font Awesome from vets-website

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/vets-design-system-documentation#2944]
## Description
On June 20th, we will be re-adding the eslint error for font awesome, meaning teams will need to migrate to va-icon as they work. Our original plan was to remove the font awesome dependency from vets-website on June 26th. On the 26th, we'll take a look at how many instances there are remaining, including in css files and test assertions, and remove the rest ourselves. Finally, we'll remove the font awesome dependency from vets-website's package.json file. 

## Considerations
- Most font awesome icons are used by their css class names, though some are referenced directly in stylesheets
- Some e2e and unit test assertions check for font awesome class names
- Since we're waiting to take action on this until the 26th, there's an expectation here that it could carry over into the next sprint depending on number of remaining instances. 
- May want to do this in small PRs to avoid CI issues 

## Tasks
- Replace all inline font awesome classes with va-icon
- Replace all css styles that reference font awesome with va-icon
- Update e2e / unit test assertions, or remove them if they are no longer needed and/or don't actually test anything
- Remove font awesome from the dependency list in vets-website

## Acceptance Criteria
- [ ] above tasks have been completed

**Comment by Andrew565** [comment]
Icon page of Component Usage Sheet here: https://docs.google.com/spreadsheets/d/1IKi_kNSQtasTb3hHbcKbhHLv7BCYCmRqsUtaRPcrtG8/edit?gid=2089291551#gid=2089291551

**Comment by caw310** [comment]
This ticket will take 2 sprints. The team has decided NOT to break this work down as it will be easier to track in the Google sheet that has the list of the 124 instances of FA that need to be removed. 
@it-harrison and @rmessina1010 will take point on this so any engineer who works on this ticket, please coordinate with them.

**Comment by it-harrison** [comment]
The work for this ticket is done; however we are waiting on some teams to review PRs. We are closing this ticket and opening a new one to monitor the remaining PRs.
